scripts/load_meeting_prot.py

```diff
@@ -1,4 +1,7 @@
-"""Script to load info from trello and merge with info from latest meeting protocol."""
+"""This script loada info from trello and merge with info from latest meeting 
+protocol. It then loads a new meeting protocol with the merged info devided into
+comming and ongoing deliveries. The info consists of project name,project info, 
+flow cell ids and comments."""
 
 import bcbio.google
```

Comment:
I dont think its a good idea to move the spredsheets functionality to genstat since it need to be editable. I rather think it should be replaced by trello.. @guillermo-carrasco  @vezzi